rllib/core/learner/learner.py

```diff
@@ -1646,20 +1650,22 @@ def _log_steps_trained_metrics(self, batch: MultiAgentBatch):
                 reduce="sum",
                 with_throughput=True,
             )
-            # Log module steps (sum of all modules).
-            self.metrics.log_value(
-                key=(ALL_MODULES, NUM_MODULE_STEPS_TRAINED),
-                value=module_batch_size,
-                reduce="sum",
-                clear_on_reduce=True,
-                with_throughput=True,
-            )
-            self.metrics.log_value(
-                key=(ALL_MODULES, NUM_MODULE_STEPS_TRAINED_LIFETIME),
-                value=module_batch_size,
-                reduce="sum",
-                with_throughput=True,
-            )
+            total_module_steps += module_batch_size
+
+        # Log module steps (sum of all modules).
+        self.metrics.log_value(
+            key=(ALL_MODULES, NUM_MODULE_STEPS_TRAINED),
+            value=total_module_steps,
+            reduce="sum",
+            clear_on_reduce=True,
+            with_throughput=True,
+        )
+        self.metrics.log_value(
+            key=(ALL_MODULES, NUM_MODULE_STEPS_TRAINED_LIFETIME),
+            value=total_module_steps,
+            reduce="sum",
+            with_throughput=True,
```

Comment:
`with_throughput=True,`

Curious why here we report with_throughput while in the rllib/core/learner/differentiable_learner.py we don't?